App.js

```diff
@@ -8,6 +8,9 @@ import { Ionicons } from '@expo/vector-icons';
 import AppNavigator from './navigation/AppNavigator';
 
 export default function App(props) {
+  Font.loadAsync({
```

Comment:
I think we're actually already loading this in line 46 -- it seems to make sense to do so in the `loadResourcesAsync` function that was built-in with Expo

Reply:
Also, just noticed but if we're adding `Poppins-Medium` and `Poppins-SemiBold` to the assets, it would make sense to add the load calls in this PR. Otherwise, we should get rid of them. @wangannie 

IMO we should add the `load` calls to `loadResourcesAsync` as well

Reply:
Sounds good! :)